### DISCUSSION TITLE
有没有办法直接通过返回的polygons直接生成html?我想用polygons重新识别，然后重填表格内容，现在直接换模型识别，好像有错误

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/TableStructureRec#40] [Ideas]
有没有办法直接通过返回的polygons直接生成html?我想用polygons重新识别，然后重填表格内容，现在直接换模型识别，好像有错误

**Comment by Joker1212** [comment]
目前返回的数据中包含了表格识别的polygons，包含了ocr的polygons，包含了表格识别的逻辑坐标，理论上你完全可以重新进行匹配和构建表格，参照代码示例，可以duank断点看一下返回到结构
![image](https://github.com/user-attachments/assets/3304052d-69fc-4376-8717-8a4ba40031fe)
